examples/speculative_decoding/distributed_generate/launch_multimodal.sh

```
@@ -0,0 +1,94 @@
+    exit 1
+fi
+
+mkdir -p "$OUTPUT_PATH"
+
+CONTAINER_IMAGE=${CONTAINER_IMAGE:-/lustre/fs1/portfolios/coreai/users/skierat/lmsysorg+sglang+v0.5.3-cu129.sqsh}
+
+counter=$START_SHARD
+worker_pids=()
+for node in "${NODE_LIST[@]}"; do
+    echo "Processing node: $node"
+    srun --output=srun_vlm_worker_${node}.log --jobid=$JOB_ID -N 1 --ntasks=1 --ntasks-per-node=1 -w "$node" \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)